core/domain/state_domain.py

```diff
@@ -2070,15 +2073,19 @@ class SubtitledHtml(python_utils.OBJECT):
 
     def __init__(self, content_id, html):
         """Initializes a SubtitledHtml domain object. Note that initializing
-        the SubtitledHtml object does not clean the html. Before saving the
-        SubtitledHtml object, validate should be called for validation and
+        the SubtitledHtml object does not clean the html. This is because we
+        sometimes need to initalize SubtitledHtml and migrate the contained
```

Comment:
"initialize" is misspelled.

Maybe point to an example (just give the module/function name) of where this process is done.

Reply:
Done